tools/clang/tools/dxa/dxa.cpp

```diff
@@ -282,8 +281,13 @@ bool DxaContext::ExtractPart(const char *pName) {
 
   WriteBlobToFile(pContent, StringRefWide(OutputFilename),
                   DXC_CP_UTF8); // TODO: Support DefaultTextCodePage
+#ifdef _WIN32
   printf("%Iu bytes written to %s\n", pContent->GetBufferSize(),
          OutputFilename.c_str());
+#else
+  printf("%zu bytes written to %s\n", pContent->GetBufferSize(),
```

Comment:
`I` as a length specifier isn't part of the C specification, but `z` is. As a result `z` should be supported on Windows too, and I think the Windows docs indicate that.